src/main/java/com/elytradev/architecture/compat/ArchitectConverter.java

```diff
@@ -0,0 +1,276 @@
+package com.elytradev.architecture.compat;
+
+import com.elytradev.architecture.common.ArchitectureMod;
+import com.elytradev.architecture.common.shape.Shape;
+import com.elytradev.architecture.common.shape.ShapeKind;
+import com.elytradev.architecture.common.tile.TileShape;
+import li.cil.architect.api.ConverterAPI;
+import li.cil.architect.api.converter.Converter;
+import li.cil.architect.api.converter.MaterialSource;
+import li.cil.architect.api.converter.SortIndex;
+import net.minecraft.block.Block;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTBase;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.Rotation;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import net.minecraftforge.fluids.FluidStack;
+import net.minecraftforge.items.IItemHandler;
+import scala.actors.threadpool.Arrays;
+
+import java.util.Collections;
+import java.util.UUID;
+import java.util.function.Predicate;
+
+/**
+ * Architect integration. Ensures that the right item is pulled from the player's inventory and that
+ * rotations are applied correctly.
+ *
+ * @author Stan Hebben
```

Comment:
There's no other uses of `@author`, even though the project has had multiple contributors. No need to change documentation style for one class. You'll be marked as an author in the commit history if this gets merged.